<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Play</class>
 <widget class="QWidget" name="Play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(54, 54, 81);
</string>
  </property>
  <widget class="QPushButton" name="BACK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_CityNovaTitulRgLt</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton
{
  background-color: rgb(79, 79, 118);
border-style: dotted;
color: rgb(89, 255, 122);
}

QPushButton:pressed
{
  background-color:  rgb(39, 39, 59);    
}

QPushButton:hover:!pressed
{
  background-color: rgb(107, 107, 161);
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>500</width>
     <height>320</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whipsmart</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 235, 134);</string>
   </property>
   <property name="title">
    <string>Введи слово на английском и перевод</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="word">
       <property name="font">
        <font>
         <family>Whipsmart</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Слово</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="translate">
       <property name="font">
        <font>
         <family>Whipsmart</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Перевод</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="typed_word">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="typed_translate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>word</zorder>
    <zorder>translate</zorder>
    <zorder>typed_translate</zorder>
    <zorder>typed_word</zorder>
   </widget>
   <widget class="QPushButton" name="PLAY_MUSIC">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

border-image : url(:/icon_music.png);
color: rgb(97, 218, 255);
}

QPushButton:pressed
{
border-image :url(:/icon_music_pressed.png)
}

QPushButton:hover:!pressed
{
 border-image : url(:/icon_music.png);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CHECK">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whipsmart</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctness_indicator">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>a_CityNovaTitulRgLt</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SKIP">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Whipsmart</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Пропустить</string>
    </property>
   </widget>
   <widget class="AnimatedLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>a_CityNovaTitulRgLt</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>PLAY_MUSIC</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>CHECK</zorder>
   <zorder>correctness_indicator</zorder>
   <zorder>SKIP</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QComboBox" name="COMBO_BOX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 79, 118);
border-style: dotted;
color: rgb(89, 255, 122)</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedLabel</class>
   <extends>QLabel</extends>
   <header>animatedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
